projects/ngxpert/hot-toast/src/lib/components/hot-toast-group-item/hot-toast-group-item.component.ts

```diff
@@ -159,18 +160,18 @@ export class HotToastGroupItemComponent {
   }
 
   ngOnInit() {
-    if (isTemplateRef(this.toast.message)) {
-      this.context = { $implicit: this.toastRef };
+    if (isTemplateRef(this.toast().message)) {
+      this.context = { $implicit: this.toastRef() };
```

Comment:
Make it a computed property.

Reply:
Do you mean this ?
    
    const isTempRef = computed(() => isTemplateRef(this.toast().message));